packages/ia-components/components/audio-players/archive-audio-jwplayer-wrapper.jsx

```diff
@@ -85,6 +85,14 @@ class ArchiveAudioPlayer extends Component {
           this.state.jwplayerInstance.on('paused', () => {
             this.setState({ trackStarting: false, playerEverStarted: true });
           });
+
+          this.state.jwplayerInstance.on('play', () => {
```

Comment:
was hesitant to listening directly to jwp events, but play8 doesn't have a current event that matches "player starting" event.  this closes loop on knowing "when" current player is triggered by patron.